xiangqi updated/src/LazyComponent/LazyLoading.js

```diff
@@ -0,0 +1,38 @@
+import React, { lazy } from 'react';
+import { Bars } from 'react-loading-icons';
+
+const Home = lazy(() => import('../Pages/Home'));
+const SignIn = lazy(() => import('../Pages/SignIn'));
+const SignUp = lazy(() => import('../Pages/SignUp'));
+const XiangqiBoard = lazy(() => import('../Pages/XiangqiBoard'));
+const LandingPage = lazy(()=> import('../Pages/LandingPage.js'));
+
+export const LazyHome = () => (
+  <React.Suspense fallback={<div><Bars /></div>}>
+    <Home />
+  </React.Suspense>
+);
+
+export const LazySignIn = () => (
+  <React.Suspense fallback={<div><Bars /></div>}>
+    <SignIn />
+  </React.Suspense>
+);
+
+export const LazySignUp = () => (
+  <React.Suspense fallback={<div><Bars /></div>}>
+    <SignUp />
+  </React.Suspense>
+);
+
+export const LazyXiangqiBoard = () => (
+  <React.Suspense fallback={<div><Bars /></div>}>
+    <XiangqiBoard />
+  </React.Suspense>
+);
+
+export const LazyLandingPage = () => (
+  <React.Suspense fallback={<div><Bars /></div>}>
+    <LandingPage />
+  </React.Suspense>
+);
```

Comment:
- Try to use absolute imports in most of the cases !
- Can we do Instead of:
   `<React.Suspense fallback={<div><Bars /></div>}>`
   Do this
   `<React.Suspense fallback={<Bars />}>`